x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -80,11 +97,17 @@ public DenseVectorFieldMapper build(BuilderContext context) {
         @Override
         public Mapper.Builder<?,?> parse(String name, Map<String, Object> node, ParserContext parserContext) throws MapperParsingException {
             DenseVectorFieldMapper.Builder builder = new DenseVectorFieldMapper.Builder(name);
-            return builder;
+            Object dimsField = node.remove("dims");
+            if (dimsField == null) {
```

Comment:
Using the same reasoning as above, it would be nice to move this check to `DenseVectorFieldMapper.Builder#build`.

Reply:
@jtibshirani Thanks, but we need this check here to ensure  we don't get `NullPointerException` on the line that follows after that:

```
int dims = XContentMapValues.nodeIntegerValue(dimsField);
```
Also, I was thinking that `public Builder dims(int dims)` can accept `dims` parameter as a primitive integer.

Reply:
👍

Reply:
One way to do this is to check if the node is null, and only parse it and call `Builder#dims` if it is not null. Anyways, a small comment, the change looks good to me!